src/Components/Pages/Details/InvoiceDetails.tsx

```diff
@@ -176,28 +159,47 @@ export const InvoiceDetails: FunctionComponent<InvoiceDetailsProps> = ({ }) => {
                     <div>Price</div>
                     <div>Amount</div>
                 </InvoiceLineItemHeader>
-                {invoice.getLineItemsList().map((lineItem, i) => <InvoiceLineItem key={`lineItem-${i}`}>
+                {invoiceCalc?.lineItems?.map((lineItem: LineItemCalc, i: number) => <InvoiceLineItem key={`lineItem-${i}`}>
                     <div>
-                        <b>{lineItem.getName()}</b>
-                        <p>{lineItem.getDescription()}</p>
+                        <b>{lineItem.name}</b>
+                        <p>{lineItem.description}</p>
                     </div>
-                    <div>{lineItem.getQuantity()}</div>
-                    <div>{formatter(lineItemPrice(lineItem))}</div>
-                    <div><b>{formatter(lineItemTotal(lineItem))}</b></div>
+                    <div>{lineItem.quantity}</div>
+                    <div>{formatter(lineItem.price)}</div>
+                    <div><b>{formatter(lineItem.total)}</b></div>
                 </InvoiceLineItem>)}
             </InvoiceLineItems>
+            {invoiceCalc?.payments?.length > 0 && (
+                <InvoiceLineItems>
```

Comment:
I was thinking I should probably rename this styled component to something more generic now that I'm using it for payments, too.  Thoughts?